src/components/Calculator.js

```diff
@@ -0,0 +1,14 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calc-wrapper">
+    <div className="calc-result">
```

Comment:
- [ ] It would be better if the component `Calculator` is class-based as one of the requirements. 😄 👍